src/components/AllArticles.jsx

```diff
@@ -1,39 +1,38 @@
-import { useState, useEffect } from "react"
-import axios from "axios"
+import { useState, useEffect } from "react";
+import axios from "axios";
 
-function AllArticles(){
+function AllArticles() {
+  const [articles, setArticles] = useState([]);
```

Comment:
You could add some _loading state_ here to conditionally render something to tell a user the data is still being fetched while they are waiting ( for those on a slower internet connection ) 💡